ext/syncthing/src/github.com/syncthing/syncthing

```diff
@@ -1 +1 @@
-Subproject commit 668eb7c398c2e8d0b9f7beb319b7789681d7d89f
+Subproject commit 6d280e7b6461d407c90f8706e348393c657e995f
```

Comment:
I did not intend to change something there. Besides my changes in the java-classes, I only rebased on your master - could the change come from there?